src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1

```diff
@@ -460,10 +461,24 @@ class ADOSVTBase: SVTBase {
 			#added check azuretable check here, if ((azuretable is used for storing bug info and scan mode is CA) OR azuretable bug info is disabed) then only allow bug logging
 			$scanSource = [AzSKSettings]::GetInstance().GetScanSource();
 			$isAzureTableEnabled = [Helpers]::CheckMember($this.ControlSettings.BugLogging, "UseAzureStorageAccount");
+			
+			#if bug logging is enabled for inactive resources, then only bug will be logged for inactive resources.
+			if([Helpers]::CheckMember($this.ControlSettings.BugLogging, "BugLogForInactiveResources"))
```

Comment:
check member will only check whether property exists or not. we should also check whether it is enabled or not. Also t should be renamed to "LogBugsForInactiveResources".

Reply:
changes done